digdag-server/src/main/java/io/digdag/server/ServerConfig.java

```diff
@@ -20,25 +21,33 @@
 @JsonSerialize(as = ImmutableServerConfig.class)
 @JsonDeserialize(as = ImmutableServerConfig.class)
 public interface ServerConfig
+    extends UndertowServerConfig
```

Comment:
Should the fact that digdag server uses undertow be part of the public interface?